zkevm-circuits/src/state_circuit/lexicographic_ordering.rs

```diff
@@ -60,8 +60,8 @@ use std::ops::Mul;
 
 // Packing the field into 480 bits:
 //   4 bits for tag,
-// + 4 bits for field_tag // TODO: this actually needs 5 bits. Either reduce id
-// + 24 bits for id       // to 23 bits, or add diff_3 etc.
+// + 5 bits for field_tag
+// + 24 bits for id
```

Comment:
23, right?

Reply:
right,  will update !! thanks

Reply:
updated in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/523/commits/f8314b0ebe9246744344c56f02c89ab327ac83d3 thanks !